test/verify/check-machines-dbus

```diff
@@ -296,6 +318,12 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Check changed boot order have persisted
         b.wait_text_not("#vm-VmNotInstalled-boot-order", bootOrder)
 
+        # Check firmware changes persisted
+        if m.image in ['fedora-31', 'fedora-testing', 'debian-testing']:
```

Comment:
Repeating old comment: Please remember the "does this support EFI" check from above in a boolean, and re-use it here. Otherwise we need to update the list in two different places.

Reply:
Done